ui/ui-components/client/client.ts

```diff
@@ -147,7 +158,7 @@ export class Client {
         const uploadPercentage = Math.round(
           (progressEvent.loaded / progressEvent.total) * 100
         );
-        return uploadHandler ? uploadHandler.set({ uploadPercentage }) : null;
+        return options.uploadHandler.set({ uploadPercentage });
```

Comment:
I put a note on the story that we can get rid of the uploadHandler entirely because we no longer upload files. It would also be fine to save it for another change.